common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/RemoteBlockPushResolver.java

```diff
@@ -1756,6 +1759,8 @@ public static class AppShufflePartitionInfo {
       this.dataFilePos = 0;
       this.mapTracker = new RoaringBitmap();
       this.chunkTracker = new RoaringBitmap();
+      CLEANER.register(this, new ResourceCleaner(dataFile, dataChannel, indexFile,
+              metaFile, appAttemptShuffleMergeId, reduceId));
```

Comment:
Indentation: 2 spaces

Reply:
change closeAllFilesAndDeleteIfNeeded to a static method, then use it.  is it ok?

Reply:
Can you submit a version for us to review together ：）

Reply:
Updated, please help me review it.